pkgs/development/tools/qtcreator/default.nix

```diff
@@ -34,6 +50,29 @@ stdenv.mkDerivation rec {
   preConfigure = ''
     substituteInPlace src/plugins/plugins.pro \
       --replace '$$[QT_INSTALL_QML]/QtQuick/Controls' '${qtquickcontrols}/${qtbase.qtQmlPrefix}/QtQuick/Controls'
+
+    # Fix paths for llvm/clang includes directories.
```

Comment:
Would using upstream patch that you mentioned work instead? If not that's not a problem but it'll be cleaner.

Reply:
Ok, but you commented in wrong place, fix for include directories still needed. Patches for correct clazy plugin name is now fetched from upstream.

Reply:
Ah yes, was commenting from phone.